analyzer/codechecker_analyzer/arg.py

```diff
@@ -0,0 +1,49 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
+"""Argument parsing related helper classes and functions."""
+
+import argparse
```

Comment:
Missing imports from `__future__`:
```suggestion
from __future__ import print_function
from __future__ import division
from __future__ import absolute_import

import argparse
```